ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-tracking.j2

```diff
@@ -12,19 +12,31 @@
 
 #### Flow Tracking Sampled
 
-| Sample Size | Minimum Sample Size | Hardware Offload for IPv4 | Hardware Offload for IPv6 |
-| ----------- | ------------------- | ------------------------- | ------------------------- |
+| Sample Size | Minimum Sample Size | Hardware Offload for IPv4 | Hardware Offload for IPv6 | Encapsulation for IPv4 | Encapsulation for IPv6 | Encapsulation for MPLS |
```

Comment:
I think a single field can work here, and then just add them as comma separated strings like `ipv4, mpls` and default to `-`.
```suggestion
| Sample Size | Minimum Sample Size | Hardware Offload for IPv4 | Hardware Offload for IPv6 | Encapsulations |
```

Reply:
Or just use the simpler logic similar to your CLI template.